OS/DiskOS/Programs/Disks/load.lua

```diff
@@ -1,39 +1,54 @@
+-- Load a game into memory either from a .png or .lk12 file or from the clipboard
+
 local source = select(1,...)
 
 local term = require("terminal")
 local eapi = require("Editors")
 
 local png = false
 
-if source and source ~= "@clip" and source ~= "-?" then
-  source = term.resolve(source)
-  if source:sub(-4,-1) == ".png" then
-    png = true
-  elseif source:sub(-5,-1) ~= ".lk12" then
-    local lksrc = source..".lk12"
-    if fs.exists(lksrc) then
-      source = lksrc
-    elseif fs.exists(source..".png") then
-      source = source..".png"
+local lk12Data = nil
```

Comment:
```suggestion
local lk12Data
```
No need to specify `= nil`, it's nil by default.

Reply:
Fixed in commit c037f254cfac11549abd3b9d474645550e955d97.